po/id_ID.po

```diff
@@ -0,0 +1,261 @@
+msgstr "Tampilkan ekstensi pengguna dan sistem secara terpisah"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:57
+msgid "Performance and stability"
+msgstr "Performa dan stabilitas"
+
+#: data/com.mattjakeman.ExtensionManager.gschema.xml:11
+msgid "Colour Scheme Preference"
+msgstr "Preferensi Skema Warna"
+
+#: data/com.mattjakeman.ExtensionManager.gschema.xml:12
+msgid ""
+"Whether to follow the system colour scheme or force either light or dark "
+"mode."
+msgstr ""
+"Apakah akan mengikuti skema warna sistem atau memaksa mode terang atau "
+"gelap."
+
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/exm-application.c:112
+msgid "translator-credits"
+msgstr "neko <smurfkandy@tutanota.com>"
+
+#: src/exm-application.c:113
+msgid ""
+"A very simple tool for browsing, downloading, and managing GNOME shell "
+"extensions."
+msgstr ""
+"Sebuah alat yang sangat sederhana untuk menjelajah, mengunduh, dan "
+"mengelola ekstensi shell GNOME."
+
+#: src/exm-application.c:119
+msgid "Project Homepage"
+msgstr "Beranda Proyek"
+
+#: src/exm-browse-page.blp:7
+msgid "Search for extensions"
+msgstr "Cari ekstensi"
+
+#: src/exm-browse-page.blp:8
+msgid ""
+"Enter a keyword to search 'extensions.gnome.org' for GNOME Shell Extensions."
+msgstr ""
+"Masukan kata kunci untuk mencari 'extensions.gnome.org' untuk Ekstensi "
+"Shell GNOME."
+
+#: src/exm-browse-page.blp:22
+msgid "e.g. \"Blur my Shell\""
+msgstr "contoh \"Blur my Shell\""
+
+#. Translators: dropdown items for sorting search results
+#: src/exm-browse-page.blp:33
+msgid "Popularity"
+msgstr "Kepopuleran"
+
+#: src/exm-browse-page.blp:33
+msgid "Downloads"
+msgstr "Unduhan"
+
+#: src/exm-browse-page.blp:33
+msgid "Recent"
+msgstr "Terkini"
+
+#: src/exm-browse-page.blp:33
+msgid "Name"
+msgstr "Nama"
+
+#: src/exm-browse-page.blp:60
+msgid "No Results Found"
+msgstr "Tidak Ada Hasil Yang Ditemukan"
+
+#: src/exm-detail-view.blp:35 src/exm-detail-view.c:287
+msgid "An Error Occurred"
+msgstr "Sebuah Error Terjadi"
+
+#: src/exm-detail-view.blp:95
+msgid "Description"
+msgstr "Deskripsi"
+
+#: src/exm-detail-view.blp:113
+msgid "Supported Versions"
+msgstr "Versi Yang Didukung"
+
+#: src/exm-detail-view.blp:129
+msgid "Links"
+msgstr "Tautan"
+
+#: src/exm-detail-view.blp:139
+msgid "View on Extensions"
+msgstr "Lihat dalam Extensions"
+
+#: src/exm-detail-view.c:120
+msgid "This extension is incompatible with your current version of GNOME."
+msgstr "Ekstensi ini tidak cocok untuk versi GNOME kamu."
+
+#: src/exm-detail-view.c:129 src/exm-search-row.blp:75
+msgid "Install"
+msgstr "Pasang"
+
+#: src/exm-detail-view.c:134 src/exm-search-row.c:153 src/exm-window.blp:42
+msgid "Installed"
+msgstr "Terpasang"
+
+#: src/exm-detail-view.c:138 src/exm-search-row.c:159
+msgid "Unsupported"
+msgstr "Tidak didukung"
+
+#. Translators: Use unicode ellipsis '…' rather than three dots '...'
+#: src/exm-detail-view.c:303
+msgid "Loading…"
+msgstr "Memuat..."
```

Comment:
```suggestion
msgstr "Memuat…"
```
Ah, I missed this one. We should be using the unicode '…' instead of three dots '...' for consistency.